### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by francisuloko**
Summary:

In this project, I have implemented the following:

- Used React components.
- Used React props.
- Used React Router.
- Connected React and Redux.
- Handled events in a React app.
- Wrote integration tests with a React testing library.
- Used styles in a React app.
- Used React life cycle methods.
- Applied React best practices and language style guides in code.
- Used store, actions and reducers in React.

View Project Presenation [Here](https://www.loom.com/share/7892ec5678a54497aa21971995655f1e).

### CODE REVIEW

File: src/components/World/World.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                {
+                  covidTotal.today_recovered
+                }
+              </span>
+            </div>
+          </div>
+          <div className="row m-0 p-0">
+            <div className="col-12 themed-grid-col text-start text-secondary">
+              Filtered by Country
+            </div>
```
Comment (Georjane) [inline src/components/World/World.js:47]: - [ ] Could you please add a filter functionality to this element? As per this project requirements, the Home page should have a list of categories that could be filtered using given parameters, e.g. category name. See [this example](https://the-catalystmc.github.io/crypto-market-cap-app/#/currencies) of how the filter component has being implemented.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  background-color: #634e46ff;
+}
+
+.grid-item:nth-child(odd) {
+  background-color: #d61a3c;
+}
+
+.color-2 {
+  background-color: #634e46ff;
+}
```
Comment (Georjane) [inline src/App.css:31]: - [ ] You should kindly select one main color for the app and be consistent (it does **not** need to be the same color as in the design guidelines). I think you have two different colors on your App

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, Team <STUDENTS>, ✋🏽 

Good job so far! You have a very professional Readme file, your PR is professional and you have a good mastery of git-flow. Your linters are all passing, I applaud that too.
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️

- [ ] Please do delete empty files such as the Header.js file

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding more test cases to also cover the negative scenarios

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @francisuloko  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by francisuloko** [comment]
Hi guys, thank you so much for reviewing and approving my code.
Happy Coding :computer:
@Georjane @mricanho
